index.html

```diff
@@ -1,3 +1,130 @@
+  <meta property="og:site_name" content="gazr.io">
+  <meta property="og:description" content="Opinionated specification to launch common goals on your project (tests, checkstyle, format, complexity ...).">
+  <meta property="og:type" content="website">
+  <!-- Google Tag Manager 
+  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  })(window,document,'script','dataLayer','GTM-PJW7PJJ');</script>
+  -->
+  <!-- End Google Tag Manager -->
+</head>
+<body>
+  <!-- Google Tag Manager (noscript) 
+  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PJW7PJJ"
+  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+  -->
+  <!-- End Google Tag Manager (noscript) -->
+    <header>
+        <img class="avatar" src="/images/gazr.svg" alt="Gazr">
+        <p><strong>gazr</strong>, pronounced "Gather", is an opinionated way to launch <strong>common goals</strong> on your project.</p>  
+    </header>
+    <section>
+        <div class="box">
+            <h2 class="main-title">Main principles</h2>
+            <div class="box-content">
+                <ul>
+                    <li>
+                        A Container runtime as the <strong>only one</strong> dependency in developer laptop.
+                        <p class="details">
+                            Docker provides a full integration with <a href="https://www.docker.com/docker-mac">Mac</a>, <a href="https://www.docker.com/docker-windows">Windows</a> & <a href="https://www.docker.com/docker-ubuntu">Linux <em>(ubuntu)</em></a>.
+                        </p>
+                    </li>
+                    <li>
+                        Use generic tasks name to launch common actions.
+                    </li>
+                    <li>
+                        Let developers use the technologies & library they want.
+                    </li>
+                    <li>
+                        Simplify technical on-boarding.
+                    </li>
+                    <li>
+                        Simplify projects switching.
+                    </li>
+                </ul>
+            </div>
+        </div>
+        <div class="box box-grey">
+            <h2 class="main-title"><em>Makefile</em> specification</h2>
+            <ul class="make-specification">
+                <li>
+                    <span>make checkstyle</span>
+                    <p class="details">
+                        Check lint, checkstyle rules. e.g. <a href="https://www.pylint.org/">pylint</a>, <a href="https://github.com/squizlabs/PHP_CodeSniffer">phpcs</a>, <a href="https://eslint.org/">eslint</a>, <a href="https://github.com/checkstyle/checkstyle">checkstyle <em>(java)</em></a> etc ... 
```

Comment:
code styling rules (not checkstyle, it's not an english word)

Reply:
Fixed